packages/plugin-http-client/src/rest/mapper/__tests__/BaseMapperSpec.js

```diff
@@ -0,0 +1,30 @@
+import { BaseMapper } from '../BaseMapper';
+
+describe('BaseMapper createMapperItem', () => {
+  it('should return MapperItem for MapperItem as value', () => {
```

Comment:
In every test you use 'fieldKey' and 'newKey'. It would be nice to have different string in those tests. Right now it is really confusing what each test is testing.

Reply:
Snad jsem pochopil a upravil.